projects/addon-doc/utils/is-page-group.ts

```diff
@@ -0,0 +1,5 @@
+import {TuiDocPage, TuiDocPageGroup} from '@taiga-ui/addon-doc/interfaces';
+
+export const isPageGroup = (
+    page: TuiDocPage | TuiDocPageGroup,
+): page is TuiDocPageGroup => `subPages` in page;
```

Comment:
1. Please use regular function instead of arrow function. It's needed for naming convention when use exported regular function 
2. `isPageGroup` -> `tuiIsPageGroup`